source/includes/_cluster.md

```diff
@@ -0,0 +1,419 @@
+# Cluster
+
+## The ClusterState Resource
+
+<aside class="notice">
+This section describes API features that are part of the Cloudify premium edition
+</aside>
+
+```python
+# include this code when using cloudify python client-
+from cloudify_rest_client import CloudifyClient
+client = CloudifyClient(
+        host='<manager-ip>',
+        username='<manager-username>',
+        password='<manager-password>')
+```
+
+The ClusterState resource represents the current state of a Cloudify Manager cluster.
+
+### Attributes:
+
+Attribute | Type | Description
+--------- | ---- | -----------
+`initialized` | boolean | Whether this node is part of a cluster.
+`encryption_key` | string | Encryption key used for inter-cluster communication (only visible by admin users).
+`consul` | dict | Detailed state of the consul cluster being part of the manager infrastructure.
+`error` | string | Description of a fatal error that occured during cluster configuration or operation, if any.
+`logs` | list | Logs of the cluster operations on the current node.
+
+## Get Cluster State
+
+> Request Example
+
+```shell
+$ curl -u user:password "http://<manager-ip>/api/v3/cluster"
+```
+
+```python
+client.cluster.status()
+```
+
+```javascript
+var headers = {
+   'content-type': 'application/json',
+   'authorization': 'Basic ' + new Buffer(username + ':' + password).toString('base64')
+}
+
+var settings = {
+  "url": "http://<manager-ip>/api/v3/cluster/",
+  "method": "GET",
+  "headers": headers
+}
+
+$.ajax(settings).done(function (response) {
+  console.log(response);
+});
+```
+
+```html
+obsolete
+```
+
+> Response Example (cluster not initialized)
+
+```json
+{
+    "initialized": false
+}
+```
+
+> Response Example
+
+```json
+{
+    "initialized": true,
+    "consul": {
+        "leader": "172.20.0.3:8300"
+    },
+    "encryption_key": "<REDACTED>",
+    "error": null,
+    "logs": [
+        {
+            "message": "HA Cluster configuration complete",
+            "timestamp": 1485778546965628,
+            "cursor": "opaque cursor value"
+        }
+    ]
+}
+```
+
+`GET "{manager-ip}/api/v3/cluster"`
+
+Gets the current cluster state. The `logs` and `error` fields are hidden by
+default, but can be added to the response if specified using the `_include`
+query parameter.
+
+### Response
+A `ClusterState` resource.
+
+### URI Parameters
+* `since`: When including logs, fetch only logs after this cursor value.
```

Comment:
When including logs, ~fetch~ **fetches** only logs ~after~ **greater than** this cursor value.

_Not sure if "greater than" are the best words here. If you think that this isn't appropriate in this context, discuss it with me and we will come up with something better._